graylog2-web-interface/src/components/common/Select.css

```diff
@@ -8,4 +8,9 @@
 
 .select-sm .Select-input > input {
     padding: 0;
-}
\ No newline at end of file
+}
+
+:local(.increaseZIndex) .Select-menu-outer {
+    z-index: 3;
```

Comment:
This is unfortunately not good enough to make the options appear on top of the ace gutter, which has a `z-index` of 4. Could you increase the value a bit more, please? In the end I don't think increasing it is a problem, since opening the select options should be displayed on top of other elements in all cases I can think of.

Reply:
Hm, I tested that in the extended search UI which uses a `Select` component next to an Ace input as well and 3 was enough for that. Where do you encounter the problem in the UI, so I can verify the problem/the fix?

Reply:
Ah, that one does not use the gutter, which explains why it works there. :)